packages/core-contracts/contracts/utils/ERC20.sol

```diff
@@ -0,0 +1,93 @@
+//SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+import "../interfaces/IERC20.sol";
+
+abstract contract ERC20 is IERC20 {
+    string public override name;
+
```

Comment:
nit: same for the space between variables. On other contracts it was used just to separate blocks if adds readability (otherwise it reduce de lines seen in a single screen)

Reply:
this is actually enforced by the linter!